examples/move/closed_loop/sources/closed_loop.move

```diff
@@ -351,105 +343,121 @@ module closed_loop::closed_loop {
         vec_set::insert(&mut request.approvals, type_name::get<W>())
     }
 
+    // === Rule Config ===
+
+    /// Add a `Config` for a `Rule` in the `TokenPolicy`. Some rules may require
+    /// a `Config` to be set before the action can be added / used.
+    public fun add_rule_config<T, Rule: drop, Config: store>(
+        _rule: Rule,
```

Comment:
What would go wrong if we didn't include the witness here?

Reply:
It's generally safer to have a witness to store the config under a specified key<T> and to not allow overlap between different configurations. We assume that admin can make a mistake or get tricked into doing something, so an extra level of protection won't hurt.

And there's a small convenience feature - one doesn't need to specify all 3 generic parameters for this function as they're implied from arguments.